docs/Running-behind-nginx.md

```diff
@@ -117,6 +117,34 @@ Using the above, you access netdata on the backend servers, like this:
 - `http://nginx.server/netdata/server1/` to reach `backend-server1`
 - `http://nginx.server/netdata/server2/` to reach `backend-server2`
 
+### Using TLS communication
```

Comment:
Replace with the following:

### Using TLS communication

In case the Netdata web server has been [configured to use TLS](../web/server/#enabling-tls-support),
you must also encrypt the communication between Nginx and Netdata.

To enable encryption, first [enable SSL on nginx](http://nginx.org/en/docs/http/configuring_https_servers.html) and then put the following in the location section of the Nginx configuration: 
```
proxy_set_header X-Forwarded-Proto https;
proxy_pass https://localhost:19999;
```

If nginx is not configured as described here, you will probably receive the error `SSL_ERROR_RX_RECORD_TOO_LONG`.

Reply:
The next pull will have this fix, thanks!